LibSWBF2/Wrappers/CollisionMesh.h

```diff
@@ -25,14 +25,14 @@ namespace LibSWBF2::Wrappers
 		CollisionMesh(); 
 
 		coll *p_CollisionChunk = nullptr;
-		mutable List<uint32_t> m_Indicies;
+		mutable List<uint16_t> m_Indicies;
 
 
 	public:
 		
 		String GetName() const;
 
-		void GetIndexBuffer(ETopology requestedTopology, uint32_t& count, uint32_t*& indexBuffer) const;
+		void GetIndexBuffer(ETopology requestedTopology, uint32_t& count, uint16_t*& indexBuffer) const;
```

Comment:
Does Unreal require ```uint32_t``` buffers for indices?  If not, we could cut down on some space by just keeping them 16 bit.